administrator/components/com_users/tmpl/users/default_batch_footer.php

```diff
@@ -9,8 +9,6 @@
 defined('_JEXEC') or die;
 use Joomla\CMS\Language\Text;
 
-use Joomla\CMS\Language\Text;
```

Comment:
Should this one be done against the main repo? If not, then better to remove line 10 as a new line should be after the jexec test.

Reply:
It shouldn't be done against main repo, this was due to a merge that didn't conflict but didn't merge ok either. I added the newline.